include/deal.II/matrix_free/fe_evaluation_data.h

```diff
@@ -1201,7 +1201,7 @@ FEEvaluationData<dim, Number, is_face>::set_data_pointers(
 
   const unsigned int size_scratch_data =
     std::max(tensor_dofs_per_component + 1, dofs_per_component) * n_components *
-      3 +
+      4 +
```

Comment:
@kronbichler I was not sure about this one.

Reply:
I guess you refer to the space needed by the truncation process, and was it necessary to avoid errors somewhat? I did not try to follow all numbers, but I also don't think it hurts if we make this larger, so I think we can make this change.